make/app_targets.mk

```diff
@@ -23,8 +23,13 @@ APPLICATION ?= $(basename $(notdir $(PWD)))
 .PHONY: test info gclean dialyzer dialyzer_plt dclean
 
 ifndef NO_TEST_TARGET
+
+ifeq ($(findstring s,${MAKEFLAGS}),s)
+TEST_ENV = QUIET=1
```

Comment:
Why option is named Silent, but later we name it Quiet? It's shorter by a letter but maybe a little confusing?

Reply:
Good question. I renamed to `SILENT` in fde5027adbbf158736cdd6727ae1787d2ac7f3e4.